package.json

```diff
@@ -1,16 +1,18 @@
 {
   "name": "assessment-backend",
   "version": "1.0.0",
-  "main": "index.js",
+  "main": "indexCorrected.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon indexCorrected.js"
```

Comment:
This command is called when you type `node start` into the terminal. It will spin up your app by calling the `indexCorrected.js` file.